fineract-provider/build.gradle

```diff
@@ -145,6 +147,7 @@ group = 'org.apache.fineract'
 buildDir = new File(rootProject.projectDir, "../build")
 repositories {
 	jcenter()
+    mavenCentral()
```

Comment:
Are you sure this is needed? jcentral is (should be..) a superset of mavenCentral, and already have anything that's on mavenCentral.